src/main/java/com/cdancy/jenkins/rest/features/JobsApi.java

```diff
@@ -46,133 +47,165 @@
 import com.cdancy.jenkins.rest.domain.job.ProgressiveText;
 import com.cdancy.jenkins.rest.fallbacks.JenkinsFallbacks;
 import com.cdancy.jenkins.rest.filters.JenkinsAuthenticationFilter;
+import com.cdancy.jenkins.rest.filters.ScrubNullFolderParam;
 import com.cdancy.jenkins.rest.parsers.BuildNumberToInteger;
 import com.cdancy.jenkins.rest.parsers.LocationToQueueId;
 import com.cdancy.jenkins.rest.parsers.OutputToProgressiveText;
 import com.cdancy.jenkins.rest.parsers.RequestStatusParser;
+import org.jclouds.javax.annotation.Nullable;
```

Comment:
This line does not belong here, move so it is in alphabetical order with the other `org.jclouds` imports.